types/jasmine/index.d.ts

```diff
@@ -554,6 +554,7 @@ declare namespace jasmine {
         toThrowMatching(predicate: (thrown: any) => boolean): boolean;
         toBeNegativeInfinity(expectationFailOutput?: any): boolean;
         toBePositiveInfinity(expectationFailOutput?: any): boolean;
+        toBeInstanceOf(expected: Constructor): boolean;
```

Comment:
Please add documentation:

```
    /**
     * Expect the actual value to be an instance of the expected class.
     * @since 3.5.0
     */
```

Same for the definition under `ts3.1` folder.